configuration/parse.go

```diff
@@ -35,24 +42,53 @@ func Parse(args []string) (Configuration, error) {
 	flag.Var(&config.CveList, "exclude-vulnerability", "Comma separated list of CVEs to exclude")
 	flag.StringVar(&excludeVulnerabilityFilePath, "exclude-vulnerability-file", "./.nancy-ignore", "Path to a file containing newline separated CVEs to be excluded")
 
+	iqCommand := flag.NewFlagSet("iq", flag.ExitOnError)
+	iqCommand.StringVar(&config.User, "user", "admin", "Specify username for request")
+	iqCommand.StringVar(&config.Token, "token", "admin123", "Specify token/password for request")
```

Comment:
those defaults feel kind of gross

Reply:
I can remove em, mostly just had em for like OOB use

Reply:
Yeah, I get it...

Just can't shake the "encouraging people to use default user/pass" vibe

Reply:
I'll probably write some helper warn function that if it sees defaults informs you that you are WILDIN out. WDYT?

Reply:
I can get down with that.
Just want to make sure we're instilling a sense of shame in the user

Reply:
>those defaults feel kind of gross

To clarify, it's really just the default password that feels gross

Reply:
If someone is using `admin:admin123` I have now implemented this type of behavior:

```
(base) 756 me:nancy (NancyWithIQ *)$ go list -m all | ./nancy iq -application testapp --server-url http://localhost:8070
2020/01/02 13:28:29 Nancy version: development
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
!!!! WARNING : You are using the default username and password for Nexus IQ, you should likely change these, and use a token. !!!!
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
..........
Wonderbar! No policy violations reported for this audit!
Report URL:  http://localhost:8070/ui/links/application/testapp/report/23fa8b51f32e4ec1aec6b9a32dd888c5
```

Reply:
Impl in: ea49c69